app/build.gradle

```diff
@@ -87,55 +66,118 @@ android {
             enableSplit = false
         }
     }
+
+    compileOptions {
+        encoding = 'UTF-8'
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+
+    dexOptions {
+        javaMaxHeapSize "2g"
+    }
+
+    dataBinding {
+        enabled = true
+    }
+
+    testOptions {
+        unitTests {
+            includeAndroidResources = true
+        }
+    }
 }
 
 dependencies {
     implementation fileTree(include: ['*.jar'], dir: 'libs')
-    implementation project(':seekarc')
     implementation project(':exoplayer-flac')
+    implementation project(':seekarc')
 
-    testImplementation 'junit:junit:4.12'
-    testImplementation 'org.robolectric:robolectric:3.5.1'
-    testImplementation 'org.robolectric:shadows-support-v4:3.3.2'
-
+    // Support libraries
+    implementation "com.android.support:design:${rootProject.ext.supportLibVersion}"
+    implementation "com.android.support:support-v4:${rootProject.ext.supportLibVersion}"
+    implementation "com.android.support:appcompat-v7:${rootProject.ext.supportLibVersion}"
+    implementation "com.android.support:cardview-v7:${rootProject.ext.supportLibVersion}"
+    implementation "com.android.support:palette-v7:${rootProject.ext.supportLibVersion}"
+    implementation "com.android.support:recyclerview-v7:${rootProject.ext.supportLibVersion}"
+    implementation "com.android.support:preference-v14:${rootProject.ext.supportLibVersion}"
+    implementation 'com.android.support.constraint:constraint-layout:1.1.3'
+
+    implementation "android.arch.persistence.room:runtime:${rootProject.ext.roomVersion}"
+    annotationProcessor "android.arch.persistence.room:compiler:${rootProject.ext.roomVersion}"
+    testImplementation "android.arch.persistence.room:testing:${rootProject.ext.roomVersion}"
+
+    /** Dagger */
+    // TODO: Update to latest version (https://github.com/google/dagger/releases)
+    implementation 'com.google.dagger:dagger:2.6.1' //2.20
+    annotationProcessor 'com.google.dagger:dagger-compiler:2.6.1' //2.20
+
+    /** ReactiveX */
+    // TODO: Migrate to RxJava2 (https://github.com/ReactiveX/RxJava) (https://github.com/ReactiveX/RxAndroid)
```

Comment:
This is gonna be a sucky migration. Let me know if you're interested in doing it 😄

There's already a ticket for this in the Pivotal Tracker page ([#146218779](https://www.pivotaltracker.com/story/show/146218779)), so let's just delete this TODO.

Reply:
I'd do it if I knew how lol